gb2260/division.py

```diff
@@ -61,6 +61,21 @@ def get(cls, code, year=None):
 
         raise ValueError('%r is not valid division code' % code)
 
+    @classmethod
+    def search(cls, code):
+        """Searches administrative division by its code in all revision.
+
+        :param code: The division code.
+        :returns: A :class:`gb2260.Division` object or ``None``.
+        """
+        # sorts from latest to oldest, and ``None`` means latest
+        pairs = sorted(
+            data.items(), reverse=True,
+            key=lambda pair: pair[0] if pair[0] else 0xFFFF)
```

Comment:
It could be replaced with `sys.maxint` in Python 2 or `sys.maxsize` in Python 3. But I think the upper limit of 16bit integer is enough here.